pkgs/applications/audio/renoise/default.nix

```diff
@@ -56,11 +58,12 @@ stdenv.mkDerivation rec {
     ln -s $out/renoise $out/bin/renoise
 
     patchelf --set-interpreter $(cat $NIX_CC/nix-support/dynamic-linker) --set-rpath $out/lib $out/renoise
```

Comment:
I toughed it uses the binary from `mpg123`. If it uses a library from that package, it is better to extend the `patchelf` command above:

```
patchelf 
   --set-interpreter $(cat $NIX_CC/nix-support/dynamic-linker) \
   --set-rpath $out/lib:${mpg123}/lib
   $out/renoise
```

Reply:
True. I checked on their forum and it does need libmpg123.so only.
I tested locally and setting the RPATH to include `${mpg123}/lib` does the trick
(app still shows `File-IO: Enabling MP3 decoding support using system mpg123 library...` in its startup logs)

I'll correct this.

Reply:
Fixed in af31c4ef4c9b374ef7538179601c0b827e9a6efc